src/transformers/models/auto/tokenization_auto.py

```diff
@@ -59,6 +59,7 @@
                 ),
             ),
             ("align", ("BertTokenizer", "BertTokenizerFast" if is_tokenizers_available() else None)),
+            ("bark", ("BertTokenizer", "BertTokenizerFast" if is_tokenizers_available() else None)),
```

Comment:
Since we don't use `Processor` in the current version of the pipeline, we need to instantiate the TTS pipeline with the TTS model tokenizer.

This means that the model tokenizer must be called with AutoTokenizer.

Reply:
It's good to have defaults anyway!